steps/aws-secrets-manager/1.0.0/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Fetch secrets
+summary: Fetch secrets from AWS Secrets Manager
+description: |
+  The main intention of this custom step is to allow storage of sensitive data on AWS Secrets Manager, and fetching them on-demand during Bitrise workflow execution.
+
+  When a workflow runs, the step fetches a list of specified secrets from AWS Secrets Manager.
+
+  The fetched secrets are then propagated into subsequent steps in the workflow.
+
+  Checkout the main repository README for usage guide.
+website: https://github.com/MoneyLion/bitrise-step-aws-secrets-manager
+source_code_url: https://github.com/MoneyLion/bitrise-step-aws-secrets-manager
+support_url: https://github.com/MoneyLion/bitrise-step-aws-secrets-manager/issues
+published_at: 2021-07-16T23:09:29.031879+08:00
+source:
+  git: https://github.com/MoneyLion/bitrise-step-aws-secrets-manager.git
+  commit: b04005c155d9d70566f4756e16a05f2d7a569a39
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: moneylion.com/security/bitrise-step-aws-secrets-manager
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+inputs:
+- aws_role_arn: $SECRETS_AWS_ROLE_ARN
```

Comment:
Please set `$SECRETS_AWS_ROLE_ARN` as a default value for this input: https://devcenter.bitrise.io/contributors/create-your-own-step/#step-input-keys-and-values

Reply:
Addressed in moneylion/bitrise-step-aws-secrets-manager@f7be8840835f6c63e636a8ab2684d982d9286035.